tests/testthat/test-renderer1-knit-print.R

```diff
@@ -79,15 +79,15 @@ get_circles <- function(html=getHTML()) {
 
 get_elements <- function(id){
   
-  a<- runtime_evaluate_helper(id=id,class_name='show_hide_selector_widgets',list_num=0,return.value=TRUE)
-  b<- runtime_evaluate_helper(id=id,class_name='show_hide_selector_widgets',list_num=0,return.value=TRUE)
-  show_hide <- runtime_evaluate_helper(id=id,class_name='table.legend tr.label_variable',list_num=0,return.value=TRUE)
-  widget <- runtime_evaluate_helper(id=id,class_name='table.legend tr.label_variable',list_num=1,return.value=TRUE)
+  top_widget_show_hide<- runtime_evaluate_helper(id=id,class_name='show_hide_selector_widgets',list_num=0,return.value=TRUE)
```

Comment:
better, thanks!
even better would be to avoid the intermediate variable names altogether, if the intermediate variables are only used once?
for example consider the code below
```r
top_widget_show_hide<- runtime_evaluate_helper(something)
list(a178=top_widget_show_hide)
```
in this case the variable `top_widget_show_hide` is only used once, so the code could be simplified to below,
```r
list(a178=runtime_evaluate_helper(something))
```
by the way do you know what a178 means? I don't know what the 178 means. It would be better (easier to understand) if we could change that name to another name that does not contain numbers like "show_hide" -- is that possible? If not can you please add a comment explaining the meaning of 178 / 934?

Reply:
I changed "a178" to some other names but it did not work.

Reply:
I think a178 and b934 are the variable names created in this .Rmd file
https://github.com/animint/animint2/blob/be99e4582054c8f2655db640535f1e320878764e/inst/examples/test_knit_print.Rmd#L19

Reply:
I guess it has something to do with the data set.
If it was not documented before this PR, it is OK to leave it undocumented, but it would be good to add docs in a comment if possible.